tensorrt_llm/_torch/models/modeling_deepseekv3.py

```
@@ -640,18 +640,18 @@ def __init__(
     def get_scores(logits, e_score_correction_bias):
         scores = F.sigmoid(logits)
         scores_with_bias = scores + e_score_correction_bias
-        return scores, scores_with_bias
-
-    def noaux_tc(self, logits, e_score_correction_bias):
-        n_group = self.n_group
-
         if enable_llm_debug():
             has_nan = torch.isnan(scores_with_bias).any()
             if has_nan:
                 warnings.warn(
```

Ruff (0.14.4)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)